Parameters/AssimilationParameters.cs

```diff
@@ -0,0 +1,59 @@
+using System;
+using DCAPST.Interfaces;
+
+namespace DCAPST
+{   
+    public class AssimilationParameters
+    {
+        // These are all just dummy variables
+        public double p;
+        public double q;
+
+        public double x1;
+        public double x2;
+        public double x3;
+        public double x4;
+        public double x5;
+        public double x6;
+        public double x7;
+        public double x8;
+        public double x9;
+
+        public double m;
+        public double t;
+        public double sb;
+        public double j;
+        public double e;
+        public double R;
+
```

Comment:
This is called parameters - but it has a CalculateAssimilation Function, while there is already a AssimilationCalculator.
Public variable names need ///summary tags
Should they all be public?
Should they have better names?

Reply:
The class / function names should be fixed up, I'll come up with some better ones.

Strictly speaking, none of the calculator parameters need to be public, it just made things easier in a few ways (in part because that's how it was done originally), and looking things over, I'd have to refactor the code a little bit to make them all private (wouldn't be too much work though).

The properties in that class are all basically just storage for various values that are calculated throughout the model, so they can all be set on object initialisation, but not all of the values are described quantities with proper names / units. Some refer to expressions that simply show up multiple times in the function.

I did find rough descriptions I could use for the letters, and will add those.

x1 through 9 I couldn't find good names for, as they are placeholders which refer to different terms at different times, depending on which type of calculator is being used.